build.gradle

```diff
@@ -83,7 +83,9 @@ allprojects {
     } else {
       jcenter()
       mavenCentral()
-      maven { url  "https://consensys.bintray.com/pegasys-repo" }
+      maven { url "https://consensys.bintray.com/pegasys-repo" }
+      maven { url 'https://jitpack.io' }
```

Comment:
We don't want to add jitpack as a dependency resolver. We're working on improving the release process for Orion which should remove the need for this (you'll be able to get the snapshot from bintray)

Reply:
or maybe the 0.1.0-SNAPSHOT in bintray has what you need?

Reply:
https://bintray.com/consensys/binaries/orion

Reply:
(also this is the reason the build fails on jenkins, because jitpack isn't configured)

Reply:
Agreed, will switch over to `0.1.0-SNAPSHOT`. No jitpack should 'just work' but it's not a problem if we move over to the snapshot